auth0/v3/authentication/database.py

```diff
@@ -37,6 +42,10 @@ def login(self, client_id, username, password, connection, id_token=None,
             },
             headers={'Content-Type': 'application/json'}
         )
+        id_token = 'id_token' in result and result['id_token']
+        if id_token: 
+            self.token_verifier.verify(id_token, self.domain, client_id)
```

Comment:
issuer field should contain the protocol, so it should be `https://self.domain/`